mne/bem.py

```diff
@@ -1158,10 +1159,19 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
             rr, tris, volume_info = read_surface(surf_ws_out,
                                                  read_metadata=True)
-            volume_info.update(new_info)  # replace volume info, 'head' stays
 
-            write_surface(s, rr, tris, volume_info=volume_info,
-                          overwrite=overwrite)
+            if overwrite:
+                # replace volume info, 'head' stays
+                volume_info.update(new_info)
+                write_surface(surf_ws_out, rr, tris, volume_info=volume_info,
+                              overwrite=True)
+            else:
+                orig_surf = op.join(ws_dir,
+                                    '%s_%s_original_surface' % (subject, s))
+                # rename surfaces with the original volume info
+                os.rename(surf_ws_out, orig_surf)
```

Comment:
do we want to keep orig_surf ? is this necessary?

Reply:
That's my doubt too, since I'm not very familiar with FreeSurfer. To be conservative, I chose to rename the original surfaces if `overwrite=False`. If we do not need the original surfaces, the codes will be more compact.

Reply:
make it more compact. We'll add it back if we need them.

@larsoner agreed?

Reply:
Yes agreed

Reply:
Thanks for the comments, I've updated the codes!